src/pages/Calendar/Calendar-Day.vue

```diff
@@ -0,0 +1,393 @@
+        const addedTime = this.addTime(flowRun.start_time, 2, 'm')
+
+        flowRun.start = !flowRun.start_time
+          ? this.formatCalendarTime(flowRun.scheduled_start_time)
+          : this.formatCalendarTime(flowRun.start_time)
+
+        flowRun.end =
+          flowRun.start_time && !flowRun.end_time
+            ? this.formatCalendarTime(new Date())
+            : diff < 60000
+            ? addedTime
+            : flowRun.start_time < this.date &&
+              flowRun.end_time?.split('T')[0] > this.date
+            ? this.formatCalendarTime(new Date())
+            : this.formatCalendarTime(flowRun.end_time)
+
+        flowRun.timed =
+          flowRun.start_time < this.date
+            ? !flowRun.end_time
+              ? false
+              : flowRun.end_time.split('T')[0] > this.date
+              ? false
+              : true
+            : true
+
+        flowRun.category = flowRun.flow_id
+        return flowRun
+      })
+
+      this.intervalHeight =
+        this.type === 'day'
+          ? uniqueRuns.length < 100
+            ? 100
+            : uniqueRuns.length > 1500
+            ? uniqueRuns.length / 2
+            : uniqueRuns.length
+          : 100
+
+      this.gettingRuns = false
+
+      this.scheduleBanner =
+        this.flow?.is_schedule_active &&
+        this.upcoming.length > 8 &&
+        !this.closeBanner
+          ? true
+          : !this.closeBanner
+          ? new Date(this.date) > new Date()
+          : false
+      return updatedRuns
+    },
+    eventColor(event) {
+      return event.state ? event.state : 'primary'
+    },
+    striped(event) {
+      return event.state === 'Scheduled'
+        ? 'striped'
+        : event.state === 'Submitted'
+        ? 'darker'
+        : ''
+    },
+    handleEventClick({ nativeEvent, event }) {
+      const open = () => {
+        this.selectedEvent = event
+        this.selectedElement = nativeEvent.target
+        this.selectedOpen = true
+      }
+      if (this.selectedOpen) {
+        this.selectedOpen = false
+        setTimeout(open, 10)
+      } else {
+        open()
+      }
+    },
+    handleScheduleBanner() {
+      this.scheduleBanner = false
+      this.closeBanner = true
+    },
+    timeNow(time) {
+      return time.split(':')[0] == this.getHour(new Date()) - 2
+        ? 'scroll-here'
+        : null
+    },
+    scrollToElement() {
+      const el = this.$el.getElementsByClassName('scroll-here')[0]
+      if (el) {
+        el.scrollIntoView()
+      }
+    }
+  },
+  apollo: {
+    flow: {
+      query: require('@/graphql/Calendar/calendar-flows.gql'),
+      variables() {
+        return {
+          id: this.flowId
+        }
+      },
+      skip() {
+        return this.skip
+      },
+      fetchPolicy: 'cache-first',
+      loadingKey: 'loadingKey',
+      update: data => data.flow_by_pk
+    }
+  }
+}
+</script>
+
+<template>
+  <v-skeleton-loader
```

Comment:
I wonder if we can forgo the skeleton loader in favor of showing the calendar immediately but empty? I think it would be a little bit less jarring